everware/git_processor.py

```diff
@@ -177,3 +184,30 @@ def load_state(self, state):
             if key in state:
                 setattr(self, key, state[key])
 
+    def parse(self, everware_yml):
+        import yaml
+        with open(everware_yml) as file:
+            text = yaml.load(file)
+        volumes = []
+        for element in text:
+            volumes += text.get(element).get("volumes")
```

Comment:
let's rename it into 'data' ('volumes' is already taken)
also it should be possible to specify exact location inside container where data volume should be mapped (or use '/notebook/data' by default)